config/routes.rb

```diff
@@ -1,9 +1,12 @@
 Rails.application.routes.draw do
   root to: 'users#index'
-  resources :users, only: [:index, :new]
+  resources :users, only: [:index, :new, :create]
   resources :categories, only: [:index, :show]
 
   get '/login', to: 'sessions#new'
   post '/login', to: 'sessions#create'
   delete '/logout', to: 'sessions#destroy'
+
+  get '/profile', to: 'users#show'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.